dhis-2/dhis-services/dhis-service-administration/pom.xml

```diff
@@ -102,8 +102,13 @@
       <artifactId>commons-io</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+       <groupId>org.apache.logging.log4j</groupId>
+       <artifactId>log4j-slf4j-impl</artifactId>
+      <scope>provided</scope>
```

Comment:
in the dhis-api/pom.xml its the other way around with slf4j-api provided. why is that?

Reply:
I am not sure about this specific one.

Anyway, I think that logs in general is something that we will need to review at some point.
At least for now I managed to remove a lot of redundant logging dependencies that were coming from a lot of different dependencies.